### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by TheMeror** [victornpb/undiscord#320] [1. Questions & Answers]
![image](https://user-images.githubusercontent.com/56722006/157549027-529ab7ea-1e6f-413a-8652-f4743603b885.png)

I needed to stop the script. When I wanted to run it again, it gave an error. Messages that were very old were not deleted.

**Comment by aijorgenson** [comment]
https://github.com/victornpb/deleteDiscordMessages/pull/323
